app/models/training_module.rb

```diff
@@ -9,7 +9,7 @@ def self.load_file
   end
 
   def about_training
-    @about ||= AboutTraining.find_by(training_module: name)
+    @about_training ||= AboutTraining.find_by(training_module: name)
   end
```

Comment:
The `AboutTraining` class has been removed. So this method is now redundant.

Reply:
Method has been removed